src/index.html

```diff
@@ -5,9 +5,34 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product" data-qa="card">
+      <img
+      class="product__img" src="./images/imac.jpeg"
+      alt="imac"
+      >
+      <p class="product__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      <p class="product__code">Product code: 195434</p>
+      <div class="product__rating">
+        <div class="product__rating-stars">
+          <div class="product__rating-stars-star"></div>
```

Comment:
Stars should be a separate block. Create a new scss file for it ;)
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
```